i18n/en/docusaurus-plugin-content-docs/current/zero/zero/hardware-design/hardware-interface.md

```diff
@@ -41,7 +41,6 @@ The ZERO provides a 40 pin GPIO header that is compatible with most peer accesso
 
 **Hint: Actual compatibility is based on usage. **
 
-<div className='gpio_style'>
 
```

Comment:
If you can't see the action log, then it is the change like this. You now have 2 consecutive empty lines.